eos-tech-support/eos-download-image

```diff
@@ -315,12 +320,21 @@ def fetch_image(session, args, base):
         sys.exit(1)
 
     personality = personalities[args.personality]
-    meta = personality['full']
-    if 'boot' in personality and personality['boot']['file']:
-        download_and_verify(session, base, personality['boot'],
-                            what='boot', destdir=args.outdir)
+    if args.iso:
+        what = 'iso'
+    else:
+        what = 'full'
+        if 'boot' in personality and personality['boot']['file']:
+            download_and_verify(session, base, personality['boot'],
+                                what='boot', destdir=args.outdir)
```

Comment:
Just noticed this now, but all the other calls to `download_and_verify` are wrapped in `print` except this one.